pkg/trace/config/subscriber.go

```diff
@@ -18,11 +18,12 @@ type Subscriber struct {
 	configs        pbgo.ConfigResponse
 	once           sync.Once
 	stopSubscriber context.CancelFunc
+	tracerInfos    chan *pbgo.TracerInfo
 }
 
 // NewSubscriber returns a new configuration store
 func NewSubscriber() *Subscriber {
-	return &Subscriber{}
+	return &Subscriber{tracerInfos: make(chan *pbgo.TracerInfo, 100)}
 }
```

Comment:
Should we have the channel buffer size the same as the cache size (configured through the same property)?

Reply:
I'm not sure if it's worth it. This should not fill up and if it does, 100 sounds like a sane default to start dropping old requests.